src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const docPopulationArray = document.querySelectorAll('.population');
+
+const docTotal = document.querySelector('.total-population');
+
+const docAverage = document.querySelector('.average-population');
+
+const populationArray = [];
+
+docPopulationArray.forEach((item) => populationArray.push(item.textContent));
+
+const populationNumbers = populationArray.map(
+  (item) => +item.replaceAll(',', ''),
+);
+
+const totalPopulation = populationNumbers.reduce((total, curr) => total + curr);
+
+const averagePopulation = totalPopulation / populationArray.length;
```

Comment:
Consider checking if 'populationNumbers.length' is greater than zero before dividing by it to avoid a potential division by zero error.